.zuul.yaml

```diff
@@ -166,6 +166,92 @@
       Build STF locally and deploy from index on OCP 4.14
     nodeset: stf-crc_extracted-ocp414
 
+# ##########
+# Test jobs
+# ##########
+# Local_build
+#
+- job:
+    name: stf-crc-ocp_412-local_build-3xl
+    parent: stf-crc-ocp_412-local_build
+    nodeset:
+      nodes:
+       - name: controller
+         label: cloud-centos-9-stream-tripleo-vexxhost
```

Comment:
That's why it does not start CI jobs on other cloud provider.
Consider to change to `cloud-centos-9-stream-tripleo` so it will not force to spawn instances on Vexxhost.

Reply:
I seem to be hitting getting RETRY_LIMIT errors since I made this change